web/src/components/home-page/youtube-video.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Fragment } from 'react';
```

Comment:
I would suggest replacing/all of this with [react-player](https://github.com/CookPete/react-player).

Reply:
Thank you 🙏  I was looking for a similar library, because with a simple iframe there are some problems that can be solved by "crutches" which is not very good. Hope this fixes some issues

Reply:
I tried to redo to react-player. This package is fix the issue with width and height, but the one problem is left. It about showing youtube logo and title, I tried to research about how to hide it, but without any success - https://developers.google.com/youtube/player_parameters#august-23,-2018 . They removed ability to hide logo.

I have suggestion how to fix that, but I think it may be not good for code quality. Maybe you have any thoughts about that?

Reply:
Should probably not use YouTube for this in the future. We normally use `webm` file, delivered through our CDN. But for now, just ignore the issue with hiding logo.

Reply:
Also, I have a suggestion about video size. On small sizes video showing not good, maybe we could have two videos - for mobile screens and for wide screens?

Reply:
For example:
### **Vertical video on mobile**
<img width="415" alt="Screenshot 2021-02-03 at 16 38 13" src="https://user-images.githubusercontent.com/35150766/106762132-36e64c00-663e-11eb-928b-48a4587821f9.png">

### **Current video**
<img width="366" alt="Screenshot 2021-02-03 at 16 39 27" src="https://user-images.githubusercontent.com/35150766/106762319-639a6380-663e-11eb-9107-24b887075cbe.png">

Reply:
Sounds like a good idea!